### DISCUSSION TITLE
Stop logging to .logs

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#486] [Q&A] [answered]
Hi
When I call monaco, I always get a logfile in the directory .logs with a timestamp as name. How can I stop monaco writing this logfile?
Thanks for a hint.
Matthias

**Comment by didiladi** [comment, marked as answer]
Hi @mdicss. I'm afraid, there is currently no option in monaco to turn off this behavior. :crying_cat_face:

**Reply by mdicss** [reply]
Thank you for the answer.
So it would be a good idea to add the option to turn this off 😄
